x-pack/test_serverless/api_integration/test_suites/common/saved_objects_management/scroll_count.ts

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+// import { SavedObjectWithMetadata } from '@kbn/saved-objects-management-plugin/common';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { RoleCredentials } from '../../../../shared/services';
+
+const apiUrl = '/api/kibana/management/saved_objects/scroll/counts';
+const defaultTypes = ['visualization', 'index-pattern', 'search', 'dashboard'];
+
+export default function ({ getService }: FtrProviderContext) {
+  const svlCommonApi = getService('svlCommonApi');
+  const svlUserManager = getService('svlUserManager');
+  const supertest = getService('supertest');
+  const kibanaServer = getService('kibanaServer');
+  let roleAuthc: RoleCredentials;
+
+  describe('scroll_count', () => {
+    before(async () => {
+      roleAuthc = await svlUserManager.createApiKeyForRole('admin');
+    });
+
+    after(async () => {
+      await svlUserManager.invalidateApiKeyForRole(roleAuthc);
+    });
+    describe('scroll_count with more than 10k objects', () => {
```

Comment:
Cleaning up is expensive. I'm considering reducing this to < 2000 objects.

Reply:
Yeah it is slow. Is there a specific breakpoint that we are testing for at 10k?

If so we could reduce that breakpoint (if configurable) otherwise we should probably leave as is for now.

If there is no specific breakpoint reducing the count seems fine to me!

Reply:
>  Is there a specific breakpoint that we are testing for at 10k?

I'm not aware of one. We can leave it as-is and see how CI responds.